setup.py

```diff
@@ -74,20 +74,23 @@ def get_version(version_file):
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.10",
     # What does your project relate to?
     keywords="skeleton",
```

Comment:
I'm not sure what else would better fit here. I just know `skeleton` does not.

```suggestion
    keywords="log4j",
```